bin.src/validate_metrics_repo.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""
+Validate the contents of a RetricsRepo, by constructing all of its
+constitutent MetricSets and their Metrics.
+"""
+
+import os
+
+import lsst.pex.exceptions
+import lsst.utils
+from lsst.validate.base import MetricRepo
+
+
+def main():
+    try:
+        default_metrics_dir = os.path.join(lsst.utils.getPackageDir('validate_metrics'), 'metrics')
+    except lsst.pex.exceptions.NotFoundError:
```

Comment:
It's not what it gains us, but it's the fact that `lsst.utils.getPackageDir` raises `pex.NotFoundError`. There are plans to make the pex exceptions look like standard python exceptions (see, e.g. DM-9435), but we're stuck with that if we want to use `getPackageDir`, and I think we do.

Alternately, file a ticket against `lsst.utils` about how `getPackageDir` should raise a more standard python type and I'll back you on it.